HTML/QUIZ PAGE/index.html

```diff
@@ -33,43 +33,6 @@
           <div class="option-name">Option 4</div>
         </button>
       </div>
-      <div class="question-list">
-        <div class="question-name">What is the full form of CSS ?</div>
-        <button class="option option1">
```

Comment:
Suggestion:- you can add your options inside the `ul` and `li` elements  for better accessibility 
(don't do it if it affects the css stylings  in any way )